src/typed-method-types/apps.ts

```diff
@@ -183,6 +186,31 @@ type AppsAuthExternalGet = {
   (args: AppsAuthExternalGetArgs): Promise<AppsAuthExternalGetResponse>;
 };
 
+// AppsAuthExternalDelete
+
+type AppsAuthExternalDeleteArgs = {
+  /** @description The id of a specified external token */
+  external_token_id: string;
+};
+
+type AppsAuthExternalDeleteSuccessfulResponse = BaseResponse & {
+  ok: true;
+};
+
+type AppsAuthExternalDeleteFailedResponse = BaseResponse & {
+  ok: false;
+  // deno-lint-ignore no-explicit-any
+  [otherOptions: string]: any;
```

Comment:
This allows for the response to, essentially, be anything. Is that intentional?

Reply:
I assumed there are for the additional errors/warning strings. is it not the case? 🤔

Reply:
As long as that is acceptable I am OK with it. If we know the shape of error responses coming back, it would be better to type them appropriately, but if we don't, then maybe this is the best we can do. I would like to hear @shapirone's opinion on this though.

Reply:
We commonly use this pattern so that if we change the response of an API on the BE, the API client isn't broken/unusable until a new version updates the expected response type.

Reply:
OK cool! Resolving